### DISCUSSION TITLE
Cannot convert undefined or null to object

### DISCUSSION THREAD

**Discussion by parsagholipour** [orestbida/cookieconsent#605] [Q&A] [answered]
I followed this https://cookieconsent.orestbida.com/essential/getting-started.html#html
except I add the script runtime using js.
```html
<script src="/js/initCookieConsent.js" type="module"></script>
```
```js
import 'https://cdn.jsdelivr.net/gh/orestbida/cookieconsent@v3.0.0-rc.17/dist/cookieconsent.umd.js';

CookieConsent.run({
  // your config. goes here (required)
});
```

I get the error:
```
Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at S (cookieconsent.umd.js:7:1039)
    at cookieconsent.umd.js:7:19607
    at e.run (cookieconsent.umd.js:7:21149)
    at initCookieConsent.js:3:15
```

How should I fix it?

**Comment by orestbida** [comment, marked as answer]
You need to provide a valid [configuration](https://cookieconsent.orestbida.com/essential/getting-started.html#configuration) object. You can also obtain a configuration from the [playground](https://playground.cookieconsent.orestbida.com/), by clicking the green download button.
